gh-109812: Fix phrasing for `collections.Counter` (#109813)
python/cpython: Doc/library/collections.rst

    list(c)                         # list unique elements
    set(c)                          # convert to a set
    dict(c)                         # convert to a regular dictionary

>     c.items()                       # create a view of the dictionary’s items (elem, cnt) pairs

Author: Maybe this is more in line with the rest of the examples
```suggestion
    list(c.items())                 # convert to a list of (elem, cnt) pairs
```

Reviewer: I disagree.  The doc for c.items() should say what c.items returns.

>     c.items()                       # convert to a view object of the dictionary’s items (elem, cnt) pairs

Reviewer 2: Maybe just

```suggestion
    c.items()                       # convert to a iterator of (elem, cnt) pairs
```

?

Reviewer 3: It's not an iterator.

I'd just change "list" to "view". And "convert to" to "create", since the view doesn't really contain the data, so "convert to" seems wrong to me.

Reviewer 2: Yes, it is just an iterable 👍 

```
>>> {}.items().__iter__
<method-wrapper '__iter__' of dict_items object at 0x102cd1250>
>>> {}.items().__next__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'dict_items' object has no attribute '__next__'. Did you mean: '__ne__'?
```
